gogettr/capabilities/comments.py

```diff
@@ -0,0 +1,39 @@
+"""
+Pull comments from a specific Gettr post
+@author Konrad Iturbe
+"""
+from typing import Iterator
+
+from gogettr.capabilities.base import Capability
+from gogettr.utils import merge
+
+
+class Comments(Capability):
+    """
+    @param post_id: comment ID from URL slug, as string, eg: "p2vhax"
+    @param max: maximum number of comment posts to pull
+    """
+
+    def pull(self, post_id: str, max: int = None) -> Iterator[dict]:
+
```

Comment:
```suggestion
    def pull(self, post_id: str, max: int = None) -> Iterator[dict]:
        """
        Pull comments on a specific post.
        
        :param str post_id: ID of the post to pull comments from (e.g., "p2vhax")
        :param int max: maximum number of comment posts to pull
        """
```

Since you're providing function-level parameter information, I think this works better as a function docstring (as opposed to being at the class level).